socialhome/streams/app/tests/components/StreamElement.tests.js

```diff
@@ -99,6 +132,48 @@ describe("StreamElement", () => {
                     }).then(() => {
                         target.instance().$data.showSharesBox.should.be.false
                         target.instance().$data._sharesCount.should.eq(propsData.sharesCount + 1)
+                        target.instance().$data._hasShared.should.be.true
+                        done()
+                    })
+                })
+            })
+        })
+
+        describe("unshare", () => {
+            beforeEach(() => {
+                Vue.prototype.$http = Axios.create({
+                    xsrfCookieName: "csrftoken",
+                    xsrfHeaderName: "X-CSRFToken",
+                })
+                moxios.install(Vue.prototype.$http)
+            })
+
+            afterEach(() => {
+                moxios.uninstall()
+            })
+
+            it("removes share on server", (done) => {
+                let propsData = getStreamElementPropsData({isUserAuthenticated: true, hasShared: true})
+                let target = mount(StreamElement, {propsData})
+                // Ensure data
+                target.instance().expandShares()
+                target.instance().$data.showSharesBox.should.be.true
+                target.instance().$data._hasShared.should.be.true
+
+                // Actual thing we are testing - the unshare
+                target.instance().unshare()
+
+                moxios.wait(() => {
+                    let request = moxios.requests.mostRecent()
+                    request.respondWith({
+                        status: 200,
+                        response: {
+                            status: "ok",
+                        }
```

Comment:
I was thinking, maybe, simple objects could be one-liners. I'm looking in ESLint's doc for a rule that matches it.

Reply:
Sure 👍 Btw, have started thinking maybe should revert the change from Airbnb that one liner objects don't have extra spaces, ie:

`{foo: "bar", bar: "foo"}`

Airbnb defines it as:

`{ foo: "bar", bar: "foo" }`

I'm starting to switch my opinion on which is clearer and leaning towards the latter, more commonly used way. Should we switch or keep as is?

Reply:
I'd rather keep as is. To distinguish from one-liner closures. Plus, it's consistent with PEP-8, I think?

Reply:
Sure!